### DISCUSSION TITLE
Screen sharing not responding well

### DISCUSSION THREAD

**Discussion by cngirwa** [LiveHelperChat/livehelperchat#1956] [Q&A]
Hello!

I tried screen sharing feature but it does not render page css and stucks. I doubt the issue is NodeJS. Can you please assist on how you hosted this https://wscboh.livehelperchat.com

regards

**Comment by PeopleInside** [comment]
Hi, if you have a paid plan you can get support:
https://livehelperchat.com/support-13c.html
or you can even try Discord chat https://discord.gg/YsZXQVh for asking help.

I'm sorry to see no replies was made here.
